app/src/main/res/values/colors.xml

```diff
@@ -47,6 +47,8 @@
     <color name="backgroundBannerSecondary">#D7D1FF</color>
     <color name="colorBackgroundNotification">#404040</color>
 
+    <color name="secondaryGray">#6F7D8D</color>
```

Comment:
I am not sure if we have such color in our design system

Reply:
i think new feature is not correctly added according our design system
we need to aks Gena to add all colors to design system or change designs according to our design system =(

Reply:
The current history design doesn't fit the general design. 
Secondary gray color is `textIconSecondary`

Please work with designers and ask to change it. 
The history design does have wrong font as well